### DISCUSSION TITLE
Checkbox error prop

### DISCUSSION THREAD

**Discussion by carledwardfp** [mantinedev/mantine#1075] [Feature requests]
- [ ] Add error props to checkbox, the same as the other `Input` elements

**Comment by r3Fuze** [comment]
I agree that this would be nice. I was expecting errors to just work like other inputs when I first set it up, but adding a custom component to show the error is not that bad.

My usecase is a checkbox that must be checked to submit a form.

**Comment by wilfredjonathanjames** [comment]
Seems strange it's not the case. The `use-form` example even uses a terms of service checkbox without validation. That would normally have validation and display an error wouldn't it?

**Comment by pencilcheck** [comment]
error for checkbox still not showing up
